src/routes/environment/get.ts

```diff
@@ -13,7 +13,8 @@ interface Request {
 interface Response {
   environment: Environment;
   environmentRights: string[];
-  userEnvironmentSettings: UserEnvironmentSettings,
+  userEnvironmentSettings: UserEnvironmentSettings;
+  environmentLogo: string | null;
```

Comment:
**requested change, I think**

I would have imagined that the logo was part of the `Environment` model as:
```ts
// ...
logo?: FileFromServer | null,
// ...
```

Reply:
It was not about needing this info, but how the file is related to the environment, see https://github.com/withthegrid/platform/pull/2011#issuecomment-1453628465